plasmapy/diagnostics/langmuir.py

```diff
@@ -37,7 +37,7 @@ def _fit_func_double_lin_inverse(x, x0, y0, T0, Delta_T):
 
 
 class Characteristic:
-    r"""Class representing a single I-V probe characteristic for convenient
+    r"""Class representing a single :math:`I-V` probe characteristic for convenient
```

Comment:
Don't go too hard on the latex! The docstrings need to be readable primarily from the likes of ipython, pydoc and Jupyter. Sphinx is nice, but if tex doesn't add much value (here it doesn't) and make stuff less readable from plain text (here it does), skip it.